app/views/devise/passwords/edit.html.haml

```diff
@@ -0,0 +1,22 @@
+= render "devise/shared/error_messages", resource: resource
```

Comment:
You need to rename these to be not `.html.haml`

Reply:
I can do that!